examples/quickstart/run.py

```diff
@@ -128,125 +75,27 @@ def main(
-                and test_dataset_version_name is not None
-            )
-            train_dataset_artifact_version = client.get_artifact_version(
-                train_dataset_name, train_dataset_version_name
-            )
-            # If train dataset is specified, test dataset must be specified
-            test_dataset_artifact_version = client.get_artifact_version(
-                test_dataset_name, test_dataset_version_name
-            )
-            # Use versioned artifacts
-            run_args_train["train_dataset_id"] = (
-                train_dataset_artifact_version.id
-            )
-            run_args_train["test_dataset_id"] = (
-                test_dataset_artifact_version.id
-            )
-
-        # Run the SGD pipeline
-        pipeline_args = {}
-        if no_cache:
-            pipeline_args["enable_cache"] = False
-        pipeline_args["config_path"] = os.path.join(
-            config_folder, "training_sgd.yaml"
-        )
-        training.with_options(**pipeline_args)(**run_args_train)
-        logger.info("Training pipeline with SGD finished successfully!\n\n")
-
-        # Run the RF pipeline
-        pipeline_args = {}
-        if no_cache:
-            pipeline_args["enable_cache"] = False
-        pipeline_args["config_path"] = os.path.join(
-            config_folder, "training_rf.yaml"
-        )
-        training.with_options(**pipeline_args)(**run_args_train)
-        logger.info("Training pipeline with RF finished successfully!\n\n")
-
-    if inference_pipeline:
-        run_args_inference = {}
-        pipeline_args = {"enable_cache": False}
-        pipeline_args["config_path"] = os.path.join(
-            config_folder, "inference.yaml"
-        )
-
-        # Configure the pipeline
-        inference_configured = inference.with_options(**pipeline_args)
-
-        # Fetch the production model
-        with open(pipeline_args["config_path"], "r") as f:
-            config = yaml.load(f, Loader=yaml.SafeLoader)
-        zenml_model = client.get_model_version(
-            config["model"]["name"], config["model"]["version"]
-        )
-        preprocess_pipeline_artifact = zenml_model.get_artifact(
-            "preprocess_pipeline"
-        )
-
-        # Use the metadata of feature engineering pipeline artifact
-        #  to get the random state and target column
-        random_state = preprocess_pipeline_artifact.run_metadata[
-            "random_state"
-        ].value
-        target = preprocess_pipeline_artifact.run_metadata["target"].value
-        run_args_inference["random_state"] = random_state
-        run_args_inference["target"] = target
-
-        # Run the pipeline
-        inference_configured(**run_args_inference)
-        logger.info("Inference pipeline finished successfully!")
+    run_args_train = {}
+
+    pipeline_args = {}
+    if no_cache:
+        pipeline_args["enable_cache"] = False
+    pipeline_args["config_path"] = os.path.join(
```

Comment:
There is no config file for the `custom` orchestration environment, which means that will fail with a `FileNotFoundError` at some point I think.